### PR TITLE
Fix request in _filterCachedFastbootCookies

### PR DESCRIPTION
**Pull request by arjansingh**
@marcoow You forgot something. :smile:

### CODE REVIEW

File: addon/services/cookies.js [hunk 1]
```diff
@@ -120,9 +120,15 @@ export default Ember.Service.extend({
   },
 
   _filterCachedFastbootCookies(fastbootCookiesCache) {
-    let request = this.get('_fastboot.request');
+    let request = this.get('_fastboot._fastbootInfo.request');
```
Comment (marcoow) [inline addon/services/cookies.js]: I guess this would break the tests as they stub `_fastboot.request`?

File: addon/services/cookies.js [hunk 2]
```diff
@@ -120,9 +120,15 @@ export default Ember.Service.extend({
...
     return _collection.reduce(fastbootCookiesCache, (acc, cookie, name) => {
       let { value, options } = cookie;
+
+      if (!options) {
+        acc[name] = value;
+        return acc;
+      }
```
Comment (marcoow) [inline addon/services/cookies.js]: I guess `options` should just default to `{}` in which case none of the below cases would apply and the cookie would be returned.

**Comment by arjansingh** [comment]
I can make those changes.

**Comment by arjansingh** [comment]
This library will also probably need to be updated to accommodate https://github.com/ember-fastboot/ember-cli-fastboot/pull/171

**Comment by arjansingh** [comment]
Changes are updated. It looks like I'll be submitting a newer PR once `ember-cli-fastboot` is updated again to use the new fastboot request protocol.

**Pull request merged**